### ISSUE TITLE
[question] how to change brain external in ml agents release14?

### ISSUE THREAD

**Issue by hyuDev** [Unity-Technologies/ml-agents#5422]
hi I use ml agents release 14 version. 
As far as I know, release 14 version  don't have to designate a brain.
When I tried to learn policy using pytorch, an error code occurred as below, and I got the answer to change the brain to external. 
unityagents.exception.UnityTimeOutException: The Unity environment took too long to respond. Make sure test_env does not need user interaction to launch and that the Academy and the external Brain(s) are attached to objects in the Scene.
But I don't know how to change. What am I supposed to do?

**Comment by andrewcoh** [comment]
Hi @hyuDev 

This means your environment is not communicating with python. The behavior type on your agent should be set to `default` and also make sure your agent has a decision requester component. If these things are alright, please send more information e.g. any console output from the editor and terminal.

**Comment by hyuDev** [comment]
Hi @andrewcoh 
This problem occurs when I want to learn with my own deep learning model, not using  ppo, sac given in the ml agents git repo.
In the current release 14 version, custom DQN seems to be impossible in addition to ppo, sac, and imitation learning. I'm a little confused, Is it right I know?
Therefore, I found old versions of projects such as mlagents toolkit v0.4, v0.6 and wanted to use them. Of course, I know it won't work in my excable Unity environment, but I thought communication would be possible if I defaulted on the player.
However, I got only the above error message. 

To sum up, my question is as follows.
1. If I want to learn through custom DQN, should I use an older version such as mlagents toolkit v0.4?
2. According to your development, the network structure that can be changed in config.yaml in Release 14 is limited. Can I change the network structure by any chance?
